paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtxTest.kt

```diff
@@ -111,6 +113,53 @@ class PaymentSheetConfigurationKtxTest {
         }
     }
 
+    private fun getConfig(eKey: String): CommonConfiguration {
+        return configuration.copy(
+            customer = PaymentSheet.CustomerConfiguration(
+                id = "cus_1",
+                ephemeralKeySecret = eKey
+            ),
+        ).asCommonConfiguration()
+    }
+
+
+    @Test
+    fun `'validate' should succeed when ephemeral key secret is of correct format`() {
+        val validEphemeralKeys = listOf(
```

Comment:
would you mind breaking out these lists into their own tests actually (here and in the other test)? Right now, if one of these tests fails because just one of the cases stops working, it will be hard to determine which test case failed -- the error will probably indicate that the test failed on e.g. line 135 but we won't be able to tell which test case failed without debugging further. Ideally, it'd be really easy to see what failed and why

Reply:
👍